app/__init__.py

```diff
@@ -0,0 +1,4 @@
+from flask import Flask

```

Comment:
I think, it' not a good plase for this action
Olease move it to logic modul, app/routes.py

Reply:
it'**s**
pla**c**e

Reply:
? 
you mean - "place it to ..."

Reply:
No, I mean you have a typo in "good place for..."